platforms/corda-ent/charts/node/templates/deployment.yaml

```diff
@@ -214,51 +228,51 @@ spec:
             memory: {{ .Values.cordaJarMx }}G
           limits:
             memory: {{ add .Values.cordaJarMx 2 }}G
-      - name: store-certs-in-vault
-        image: {{ .Values.image.initContainerName }}
-        imagePullPolicy: Always
-        env:
-          - name: VAULT_ADDR
-            value: {{ $.Values.vault.address }}
-          - name: KUBERNETES_AUTH_PATH
-            value: {{ $.Values.vault.authpath }}
-          - name: VAULT_APP_ROLE
-            value: {{ $.Values.vault.role }}
-          - name: BASE_DIR
-            value: {{ $.Values.volume.baseDir }}
-          - name: CERTS_SECRET_PREFIX
-            value: {{ .Values.vault.certsecretprefix }}
-        command: ["sh", "-c"]
-        args:
-        - |-
-          #!/usr/bin/env sh
+      # - name: store-certs-in-vault
```

Comment:
remove this completely, no commented code should be checked in

Reply:
done